### PR TITLE
filter out unsupported policies

### PR DESCRIPTION
**Pull request by kickster97**
### WHAT is this pull request doing?
Adds an array `SUPPORTED_POLICIES` and filters on it `Policy`'s `merge_definitions`

### HOW can this pull request be tested?
try and add an unsupported policy, eg. HA policy, and view the effective policy definition on the queue

### CODE REVIEW

File: src/lavinmq/policy.cr [hunk 1]
```diff
@@ -10,6 +10,10 @@ module LavinMQ
   end
 
   class Policy
+    SUPPORTED_POLICIES = ["max-length", "max-length-bytes", "message-ttl", "expires", "overflow",
```
Comment (carlhoerberg) [inline src/lavinmq/policy.cr:13]: Make it a tuple, as it's static

Comment (kickster97) [reply src/lavinmq/policy.cr]: Yes, fixed!

**Review by spuun — COMMENTED** [review]
A spec would be nice!

**Comment by kickster97** [comment]
> A spec would be nice!

@spuun added a spec :)

**Review by spuun — APPROVED** [review]

**Pull request merged**